lib/entities/field-type.ts

```diff
@@ -10,6 +10,8 @@ export type FieldType =
   | { type: 'Location' }
   | { type: 'Link'; linkType: 'Asset' }
   | { type: 'Link'; linkType: 'Entry' }
+  | { type: 'ResourceLink'; linkType: 'Contentful:entry' }
```

Comment:
Q: I've seen it around being `Contentful:Entry` or is it case-insensitive - aka doesn't matter etc? 🤔